src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -29,7 +29,8 @@ const DetailDomainScreenHeader = ({
         onPress={() => {
           navigation.goBack();
         }}>
-        <ArrowLeftIcon width={20} height={12} fill="#000" />
+        {/* TODO: Garry, perlu pakai props */}
+        <ArrowLeftIcon width={20} height={12} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment indicates that props should be used for the `ArrowLeftIcon` component. However, it's not clear what kind of props are expected here. If the intention is to make the icon customizable, you might want to pass in the width and height as props.
```diff
- <ArrowLeftIcon width={20} height={12} />
+ <ArrowLeftIcon width={iconWidth} height={iconHeight} />
```
Please replace `iconWidth` and `iconHeight` with the actual prop names.



<!-- This is an auto-generated comment by OSS CodeRabbit -->